packages/vue-instantsearch/src/components/Hits.js

```diff
@@ -76,9 +83,18 @@ export default {
       );
     };
 
+    const bannerComponent = ({ banner }) => {
+      return h(Fragment, {}, [
+        bannerSlot &&
+          bannerSlot({
+            banner,
+          }),
+      ]);
+    };
```

Comment:
Instead you can just do this : 
```suggestion
    const bannerComponent =
      bannerSlot &&
      (({ banner }) =>
        bannerSlot({
          banner,
        }));
```

Reply:
Applied